tools/clang/lib/Headers/hlsl/dx/linalg.h

```diff
@@ -0,0 +1,182 @@
+// Header for linear algebra APIs.
+
+#if __spirv__
+#error "Cooperative matrix/vectors not (yet) supported for SPIRV"
```

Comment:
```suggestion
#error "Cooperative vectors not (yet) supported for SPIRV"
```
Sorry for my bad suggestion. Cooperative matrix actually is supported in SPIRV, they have a separate header for it.